utils/zsh/patches/0001-HACK-add-temp-support-for-PCRE2.patch

```diff
@@ -0,0 +1,648 @@
+From fabdb4164288b21f1323f783d257212e5aacecd4 Mon Sep 17 00:00:00 2001
```

Comment:
Regarding this patch, I think we can do it even more better.

I am not sure, why it is labeled as hack, when it seems by quick looking at this patch that it contains actually at least four commits from upstream, so we can do it as patch series as this is rather backport than hack.

Hack in my opinion is something, which is not included in upstream and won't be.

1. https://github.com/zsh-users/zsh/commit/b62e911341c8ec7446378b477c47da4256053dc0.patch
2. https://github.com/zsh-users/zsh/commit/f3f371deb376478176866fd770fbcf9bc0d0609f.patch
3. https://github.com/zsh-users/zsh/commit/b4d1c756f50909b4a13e5c8fe5f26f71e9d54f63.patch
4. https://github.com/zsh-users/zsh/commit/10bdbd8b5b0b43445aff23dcd412f25cf6aa328a.patch

Yeah, we need to remove the things, which we need, but still it will look better. :-)

Btw. Yes, I need to admit that this is a big improvement than it was, but it could be even better. 💪

Reply:
@BKPepe indeed... that was my original idea and was blocked by the pkg fixup... Took care of backporting the patch and now it's much cleaner and direct... just 5 backport patch not touched (aside from the changelog commented out)

Reply:
See @oskarirauta you should learn from this guy. :) 👏 👏 Awesome work as always.